### ISSUE TITLE
[charts] Bar chart value labels of bar should appear on top of each bar

### ISSUE THREAD

**Issue by ZaidLinkEZ** [mui/mui-x#16006]
### Steps to reproduce

The simple bar chart or bar plot should have the value labels of each to show on top of their respective bar

### Current behavior

Currently this is what i can achieve. 
![chart](https://github.com/user-attachments/assets/6c232ec9-8014-4f77-907f-e4ac7eec1a35)

### Expected behavior

I want it like this
![error](https://github.com/user-attachments/assets/7ab470d7-9965-4e61-95cc-090590b59cb5)

### Your environment

<details>
  <summary><code>npx @mui/envinfo</code></summary>

```
  Don't forget to mention which browser you used.
  Output from `npx @mui/envinfo` goes here.
```
</details>

**Search keywords**: Bar Chart, Bar Plot, Label

**Order ID**: 95670

**Comment by michelengelen** [comment]
I don't think that is possible at the moment @ZaidLinkEZ ... I'll add this as a feature request to the board! 👍🏼 

cc @alexfauquette

**Comment by ZaidLinkEZ** [comment]
> I don't think that is possible at the moment @ZaidLinkEZ ... I'll add this as a feature request to the board! 👍🏼
> 
> cc @alexfauquette

Okay, thanks for the response! Hope to see this feature soon.

**Comment by alexfauquette** [comment]
I'm closing this one as a duplicateof #12975

Don't hesitate to subscribe to the other issue, or add comments if some aspects are missing
